Dockerfile

```diff
@@ -85,6 +85,9 @@ RUN echo "Cloning nginx and building $NGINX_VERSION (rev $NGINX_COMMIT from '$NG
 
 FROM nginx:${NGINX_VERSION}
 
+HEALTHCHECK --interval=30s --timeout=3s --start-period=60s \
+  CMD (curl -f http://localhost/ipfs/QmXjYBY478Cno4jzdCcPy4NcJYFrwHZ51xaCP8vUwN9MGm/) || exit 1
```

Comment:
Wait, does this actually work? If I run `curl -f http://192.168.1.xxx:31080/ipfs/QmXjYBY478Cno4jzdCcPy4NcJYFrwHZ51xaCP8vUwN9MGm/` from CLI, it returns 23, see https://everything.curl.dev/usingcurl/returns... which would fail the HEALTCHECK, I think.

Reply:
Weird when I run this inside of the container what I see is:

```
root@DEN:/usr/src/app# curl -f http://localhost/ipfs/QmXjYBY478Cno4jzdCcPy4NcJYFrwHZ51xaCP8vUwN9MGm/
<html>
<head><title>301 Moved Permanently</title></head>
<body>
<center><h1>301 Moved Permanently</h1></center>
<hr><center>nginx</center>
</body>
</html>
root@DEN:/usr/src/app# echo $?
0

```

Reply:
Duh, yeah, please ignore that... sorry. But while looking into this, I've noticed that as-is it would actually pass (exit code 0) whether or not it "really worked" (e.g. even if it's a  wrong IPFS CID, or there is a problem to resolve it, it' still always just 301). I've had a closer look at `curl` options, what do you think about we do the following:

    curl --fail --silent --output /dev/null --location --insecure http://178.199.229.8/ipfs/QmXjYBY478Cno4jzdCcPy4NcJYFrwHZ51xaCP8vUwN9MGm

The `--fail` is (your) `-f` just perhaps more explicit in the long form.

The `--silent --output /dev/null` are with an eye towards #114.

The `--location --insecure` follows that 301 redirect - and lets use "test" that it "really worked".

Reply:
```suggestion
  CMD (curl --fail --silent --output /dev/null --location --insecure http://localhost/ipfs/QmXjYBY478Cno4jzdCcPy4NcJYFrwHZ51xaCP8vUwN9MGm/) || exit 1
```

@holdenk LGTY?

Reply:
@holdenk do you want to click Accept / Apply for this?